functions/onprem/orborus/orborus.go

```diff
@@ -104,6 +104,8 @@ var memcached = os.Getenv("SHUFFLE_MEMCACHED")
 var tenzirUrl = os.Getenv("SHUFFLE_TENZIR_URL")
 var apiKey = os.Getenv("AUTH_FOR_ORBORUS")
 
+var certPath = os.Getenv("SHUFFLE_CERT_DIR")
```

Comment:
So is there a guarantee certPath is set then? Assuming it has a default that is set before this point.